### PR TITLE
Better error message when classinfo is null

### PR DESCRIPTION
**Pull request by maxandersen**
in https://github.com/quarkusio/quarkus/issues/30254 it was found that classInfo can sometime be null in the setupdenyalljaxrs methods and we never got a reproducer to reproduce it. But the error message could at least be more informative and tell user (and us) which class is causing troubles in case this happens again.

### CODE REVIEW

File: extensions/resteasy-classic/resteasy/deployment/src/main/java/io/quarkus/resteasy/deployment/ResteasyBuiltinsProcessor.java [hunk 1]
```diff
@@ -61,6 +61,7 @@ void setUpDenyAllJaxRs(CombinedIndexBuildItem index,
             List<String> resourceClasses = resteasyDeployment.getDeployment().getScannedResourceClasses();
             for (String className : resourceClasses) {
                 ClassInfo classInfo = index.getIndex().getClassByName(DotName.createSimple(className));
+                if(classInfo==null) throw new IllegalStateException("Unable to find class info for " + className);
```
Comment (gastaldi) [inline extensions/resteasy-classic/resteasy/deployment/src/main/java/io/quarkus/resteasy/deployment/ResteasyBuiltinsProcessor.java:64]: Can we also add instructions here on what to do when this happens?

**Comment by gastaldi** [comment]
I ran `mvn process-sources` for you 😉

**Review by gastaldi — APPROVED** [review]

File: extensions/resteasy-classic/resteasy/deployment/src/main/java/io/quarkus/resteasy/deployment/ResteasyBuiltinsProcessor.java (hunk 1 above)
Comment (maxandersen) [reply extensions/resteasy-classic/resteasy/deployment/src/main/java/io/quarkus/resteasy/deployment/ResteasyBuiltinsProcessor.java]: I don't know if know what that would be?

Comment (gastaldi) [reply extensions/resteasy-classic/resteasy/deployment/src/main/java/io/quarkus/resteasy/deployment/ResteasyBuiltinsProcessor.java]: Something like: `Make sure the class exists in the application or if it's in a separate JAR make sure it's indexed or contains a beans.xml`?

Comment (maxandersen) [reply extensions/resteasy-classic/resteasy/deployment/src/main/java/io/quarkus/resteasy/deployment/ResteasyBuiltinsProcessor.java]: ah yes, that would be an improvement.

**Comment by punkratz312** [comment]
@gsmet could you please merge this in the next maintenance release. im looking forward 3.0 since last year, so it might still take some time. its a big step so no rush and no judgment or offence from my side.
if the logging will be released early i could provide new feedback. i don’t have so much time left to support. if you don’t mind, please release asap.

**Pull request merged**